test/remote_simulate.py

```diff
@@ -150,15 +150,16 @@ def setup_remote_conflux(self):
         pscp(self.options.ips_file, zipped_conf_file, "~", 3, "copy conflux configuration files to remote nodes")
         os.remove(zipped_conf_file)
 
-        self.log.info("setup conflux runtime environment ...")
-        pssh(self.options.ips_file, "tar zxf conflux_conf.tgz -C /tmp", 3, "decompress conflux configuration files")
-        pssh(self.options.ips_file, "rm conflux_conf.tgz", 3, "delete conflux configuration files")
-
-        # start conflux on all nodes
-        self.log.info("start conflux on remote nodes ...")
-        pssh(self.options.ips_file, "sh ./remote_start_conflux.sh {} {} {} > start_conflux.out".format(
+        # setup on remote nodes and start conflux
+        self.log.info("setup conflux runtime environment and start conflux on remote nodes ...")
+        cmd_kill_conflux = "killall -9 conflux || echo already killed"
+        cmd_cleanup = "rm -rf /tmp/conflux_test_*"
+        cmd_setup = "tar zxf conflux_conf.tgz -C /tmp"
+        cmd_startup = "sh ./remote_start_conflux.sh {} {} {} > start_conflux.out".format(
             self.options.tmpdir, p2p_port(0), self.options.nodes_per_host
-        ))
+        )
+        cmd = "{}; {} && {} && {}".format(cmd_kill_conflux, cmd_cleanup, cmd_setup, cmd_startup)
```

Comment:
`cleanup` may fail if it's a clean instance, so we should use `;` instead of `&&` after it?